Superfluous Disable Command Violations in the Tests

### Expected Behavior

The Run SwiftLint Build Phase passes without errors when building with the Debug configuration.

### Actual Behavior

Superfluous disable command (`// swiftlint:disable function_body_length`, `// swiftlint:disable type_body_length`) violations in the tests cause the Run SwiftLint Build Phase to fail when building with the Debug configuration.

[SwiftLint 'disable' commands are superfluous when the disabled rule would not have triggered a violation in the disabled region.](https://github.com/realm/SwiftLint/blob/0.22.0/Source/SwiftLintFramework/Rules/SuperfluousDisableCommandRule.swift)

#### Build Log

```
Loading configuration from '.swiftlint.yml'
'variable_name' rule has been renamed to 'identifier_name' and will be completely removed in a future release.
Linting Swift files in current working directory
Linting '.Package.test.swift' (1/35)
Linting 'Package.swift' (2/35)
Linting '_Resolver.swift' (3/35)
Linting 'Assembly.swift' (4/35)
Linting 'Assembler.swift' (5/35)
Linting 'Container.Logging.swift' (6/35)
Linting 'Container.swift' (7/35)
Linting 'DebugHelper.swift' (8/35)
Linting 'FunctionType.swift' (9/35)
Linting 'InstanceStorage.swift' (10/35)
Linting 'ObjectScope.Standard.swift' (11/35)
Linting 'ObjectScope.swift' (12/35)
Linting 'ServiceEntry.swift' (13/35)
Linting 'ServiceKey.swift' (14/35)
Linting 'SpinLock.swift' (15/35)
Linting 'SynchronizedResolver.swift' (16/35)
Linting 'UnavailableItems.swift' (17/35)
Linting 'LinuxMain.swift' (18/35)
Linting 'Animal.swift' (19/35)
Linting 'AssemblerSpec.swift' (20/35)
Linting 'BasicAssembly.swift' (21/35)
Linting 'Circularity.swift' (22/35)
Linting 'ContainerSpec.Arguments.swift' (23/35)
Linting 'ContainerSpec.Circularity.swift' (24/35)
Linting 'ContainerSpec.CustomScope.swift' (25/35)
Linting 'ContainerSpec.CustomStringConvertible.swift' (26/35)
Linting 'ContainerSpec.DebugHelper.swift' (27/35)
Linting 'ContainerSpec.swift' (28/35)
Linting 'Food.swift' (29/35)
Swinject/Tests/SwinjectTests/AssemblerSpec.swift:8:38: error: Superfluous Disable Command Violation: SwiftLint rule 'type_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Linting 'LoadAwareAssembly.swift' (30/35)
Swinject/Tests/SwinjectTests/ContainerSpec.Arguments.swift:8:38: error: Superfluous Disable Command Violation: SwiftLint rule 'type_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Linting 'Person.swift' (31/35)
Swinject/Tests/SwinjectTests/ContainerSpec.Circularity.swift:8:38: error: Superfluous Disable Command Violation: SwiftLint rule 'type_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Linting 'ServiceEntrySpec.swift' (32/35)
Swinject/Tests/SwinjectTests/ContainerSpec.CustomScope.swift:8:38: error: Superfluous Disable Command Violation: SwiftLint rule 'type_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Linting 'ServiceKeySpec.swift' (33/35)
Swinject/Tests/SwinjectTests/ContainerSpec.CustomStringConvertible.swift:8:38: error: Superfluous Disable Command Violation: SwiftLint rule 'type_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Linting 'SynchronizedResolverSpec.swift' (34/35)
Linting 'WeakStorageSpec.swift' (35/35)
Swinject/Tests/SwinjectTests/ContainerSpec.DebugHelper.swift:9:42: error: Superfluous Disable Command Violation: SwiftLint rule 'function_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Swinject/Tests/SwinjectTests/ContainerSpec.DebugHelper.swift:8:38: error: Superfluous Disable Command Violation: SwiftLint rule 'type_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Swinject/Tests/SwinjectTests/ServiceEntrySpec.swift:9:42: error: Superfluous Disable Command Violation: SwiftLint rule 'function_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Swinject/Tests/SwinjectTests/ServiceEntrySpec.swift:8:38: error: Superfluous Disable Command Violation: SwiftLint rule 'type_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Swinject/Tests/SwinjectTests/ServiceKeySpec.swift:8:38: error: Superfluous Disable Command Violation: SwiftLint rule 'type_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Swinject/Tests/SwinjectTests/SynchronizedResolverSpec.swift:8:38: error: Superfluous Disable Command Violation: SwiftLint rule 'type_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Swinject/Tests/SwinjectTests/WeakStorageSpec.swift:9:42: error: Superfluous Disable Command Violation: SwiftLint rule 'function_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Swinject/Tests/SwinjectTests/WeakStorageSpec.swift:8:38: error: Superfluous Disable Command Violation: SwiftLint rule 'type_body_length' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
Done linting! Found 13 violations, 13 serious in 35 files.
```

### Steps to Reproduce the Behavior

- Xcode 8.3.3
- Carthage 0.25.0
- SwiftLint 0.22.0

1. `$ git clone https://github.com/Swinject/Swinject.git`
2. `$ cd Swinject/`
3. `$ carthage update --no-use-binaries`
3. `$ open Swinject.xcodeproj`
4. Build Swinject-iOS or Swinject-OSX targets

--- a/Tests/SwinjectTests/AssemblerSpec.swift
+++ b/Tests/SwinjectTests/AssemblerSpec.swift
@@ -5,7 +5,6 @@
 //  Created by mike.owens on 12/9/15.
 //  Copyright © 2015 Swinject Contributors. All rights reserved.
 //
-// swiftlint:disable type_body_length
 // swiftlint:disable function_body_length
 
 import Foundation

--- a/Tests/SwinjectTests/ContainerSpec.Arguments.swift
+++ b/Tests/SwinjectTests/ContainerSpec.Arguments.swift
@@ -5,7 +5,6 @@
 //  Created by Yoichi Tagaya on 8/3/15.
 //  Copyright © 2015 Swinject Contributors. All rights reserved.
 //
-// swiftlint:disable type_body_length
 // swiftlint:disable function_body_length
 
 import Quick

--- a/Tests/SwinjectTests/ContainerSpec.Circularity.swift
+++ b/Tests/SwinjectTests/ContainerSpec.Circularity.swift
@@ -5,7 +5,6 @@
 //  Created by Yoichi Tagaya on 7/29/15.
 //  Copyright © 2015 Swinject Contributors. All rights reserved.
 //
-// swiftlint:disable type_body_length
 // swiftlint:disable function_body_length
 
 import Quick

--- a/Tests/SwinjectTests/ContainerSpec.CustomScope.swift
+++ b/Tests/SwinjectTests/ContainerSpec.CustomScope.swift
@@ -5,7 +5,6 @@
 //  Created by Jakub Vaňo on 11/11/16.
 //  Copyright © 2016 Swinject Contributors. All rights reserved.
 //
-// swiftlint:disable type_body_length
 // swiftlint:disable function_body_length
 
 import Quick

--- a/Tests/SwinjectTests/ContainerSpec.CustomStringConvertible.swift
+++ b/Tests/SwinjectTests/ContainerSpec.CustomStringConvertible.swift
@@ -5,7 +5,6 @@
 //  Created by Jakub Vaňo on 13/06/2017.
 //  Copyright © 2017 Swinject Contributors. All rights reserved.
 //
-// swiftlint:disable type_body_length
 // swiftlint:disable function_body_length
 // swiftlint:disable line_length
 

--- a/Tests/SwinjectTests/ContainerSpec.DebugHelper.swift
+++ b/Tests/SwinjectTests/ContainerSpec.DebugHelper.swift
@@ -5,8 +5,6 @@
 //  Created by Jakub Vaňo on 26/09/16.
 //  Copyright © 2016 Swinject Contributors. All rights reserved.
 //
-// swiftlint:disable type_body_length
-// swiftlint:disable function_body_length
 
 import Quick
 import Nimble

--- a/Tests/SwinjectTests/ServiceEntrySpec.swift
+++ b/Tests/SwinjectTests/ServiceEntrySpec.swift
@@ -5,8 +5,6 @@
 //  Created by Yoichi Tagaya on 7/29/15.
 //  Copyright © 2015 Swinject Contributors. All rights reserved.
 //
-// swiftlint:disable type_body_length
-// swiftlint:disable function_body_length
 
 import Quick
 import Nimble

--- a/Tests/SwinjectTests/ServiceKeySpec.swift
+++ b/Tests/SwinjectTests/ServiceKeySpec.swift
@@ -5,7 +5,6 @@
 //  Created by Yoichi Tagaya on 7/23/15.
 //  Copyright © 2015 Swinject Contributors. All rights reserved.
 //
-// swiftlint:disable type_body_length
 // swiftlint:disable function_body_length
 
 import Quick

--- a/Tests/SwinjectTests/SynchronizedResolverSpec.swift
+++ b/Tests/SwinjectTests/SynchronizedResolverSpec.swift
@@ -5,7 +5,6 @@
 //  Created by Yoichi Tagaya on 11/23/15.
 //  Copyright © 2015 Swinject Contributors. All rights reserved.
 //
-// swiftlint:disable type_body_length
 // swiftlint:disable function_body_length
 
 import Dispatch

--- a/Tests/SwinjectTests/WeakStorageSpec.swift
+++ b/Tests/SwinjectTests/WeakStorageSpec.swift
@@ -5,8 +5,6 @@
 //  Created by Jakub Vaňo on 11/30/16.
 //  Copyright © 2016 Swinject Contributors. All rights reserved.
 //
-// swiftlint:disable type_body_length
-// swiftlint:disable function_body_length
 
 import Quick
 import Nimble

